src/components/CaptureFilter.js

```diff
@@ -165,8 +165,12 @@ function Filter(props) {
     filter.deviceIdentifier = deviceId;
     filter.planterIdentifier = growerIdentifier;
     filter.wallet = wallet;
-    filter.dateStart = dateStart ? formatDate(dateStart) : undefined;
-    filter.dateEnd = dateEnd ? formatDate(dateEnd) : undefined;
+    filter.dateStart = dateStart
+      ? formatDate(dateStart) + 'T00:00:00.000Z'
```

Comment:
It would be great to create a common utility function that adds these suffixes to avoid duplication and make it easy to reuse in other places.

Reply:
Sure, I`ll take care of it ASAP.